bin/cqlsh.py

```diff
@@ -21,8 +21,8 @@
 import sys
 from glob import glob
 
-if sys.version_info < (3, 7):
-    sys.exit("\ncqlsh requires Python 3.7+\n")
+if sys.version_info < (3, 8) or sys.version_info >= (3, 12):
```

Comment:
12 o 11?

Reply:
12. The max supported version is Python 3.11.x, so we use `>= (3, 12)` to accept minors.